addons/website_forum/static/src/js/website_forum.js

```diff
@@ -408,10 +398,20 @@ publicWidget.registry.websiteForum = publicWidget.Widget.extend({
     _onDeleteCommentClick: function (ev) {
         ev.preventDefault();
         var $link = $(ev.currentTarget);
+        var $container = $link.closest('.o_wforum_post_comments_container');
+
         this._rpc({
             route: $link.closest('form').attr('action'),
         }).then(function () {
-            $link.parents('.comment').first().remove();
+            $link.parents('.o_wforum_post_comment').first().remove();
```

Comment:
(I know it was like that before but: `.parents(...).first()` => `.closest(...)` (except that closest checks itself too but that is not a problem))